### DISCUSSION TITLE
新艾利都上空惊现未知飞行物

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#2438] [其他]
![7913714_142435_1_lit](https://github.com/user-attachments/assets/91ec5971-31a4-41c4-81dc-9337be10db5a)
今天在二十二分街的COFF CAFE喝咖啡的时候在窗外看到了这种跟鸟一样的东西,顺手拍了下来
回家上网查了查
才知道这是旧文明一种叫“战斗机“的东西
楼主一开始还以为是空洞打捞出来的,结果翻遍了资料也没找到一模一样的"战斗机"
决定发到绳网上问问大伙近期有没有再见到这东西

**Comment by Phaethon-0019** [comment]
这还真没见过，旧文明的战斗机相关资料我翻遍了也没有，已经跟治安局那边报告了，他们说会派之前成立不久的六号小组进行警戒工作

**Comment by Enthalpy03** [comment]
这个委托我接了
